app/src/main/java/com/nervousfish/nervousfish/modules/pairing/AndroidAcceptThread.java

```diff
@@ -0,0 +1,89 @@
+package com.nervousfish.nervousfish.modules.pairing;
+
+
+import android.bluetooth.BluetoothAdapter;
+import android.bluetooth.BluetoothServerSocket;
+import android.bluetooth.BluetoothSocket;
+
+import com.nervousfish.nervousfish.events.BluetoothConnectedEvent;
+import com.nervousfish.nervousfish.modules.constants.IConstants;
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+/**
+ * This thread runs while listening for incoming connections. It behaves
+ * like a server-side client. It runs until a connection is accepted
+ * (or until cancelled).
+ */
+class AndroidAcceptThread extends Thread {
```

Comment:
Could you change this name so it is clear that it is about Bluetooth or make it more general?